### PR TITLE
RPG game

### PR DESCRIPTION
**Pull request by dacian-cioc**
Classes, Objects & Constructors homework

### CODE REVIEW

File: src/L07/homework/Enemies.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    public int getStrength() {
+        return strength;
+    }
+
+    public int getLevel() {
+        return level;
+    }
+
+    Enemies() {
+        this.health = 50;
```
Comment (KaloyanTsvetkov) [inline src/L07/homework/Enemies.java]: o	health – 150

File: src/L07/homework/Enemies.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    }
+
+    Enemies(int health, int strength, int level) {
+        this.health = health;
+        this.strength = strength;
+        this.level = level;
+
+    }
+
+    public void attack(Heroes hero) {
```
Comment (KaloyanTsvetkov) [inline src/L07/homework/Enemies.java:36]: Looks like you implemented whole battle here. Actually this shold be only a single attack.

File: src/L07/homework/Enemies.java (hunk 1 above)
Comment (dacian-cioc) [reply src/L07/homework/Enemies.java]: updated

File: src/L07/homework/Enemies.java (hunk 2 above)
Comment (dacian-cioc) [reply src/L07/homework/Enemies.java:36]: updated

**Review by KaloyanTsvetkov — APPROVED** [review]

**Pull request merged**